base/errorshow.jl

```diff
@@ -315,7 +315,7 @@ function showerror(io::IO, ex::MethodError)
             hasrows |= isrow
             push!(vec_args, isrow ? vec(arg) : arg)
         end
-        if hasrows && applicable(f, vec_args...)
+        if hasrows && applicable(f, vec_args...) && isempty(kwargs)
```

Comment:
It'd be awesome if we could do:

```julia
        if hasrows && applicable(f, vec_args...; kwargs...)
```

but looks like applicable doesn't understand kwargs.  This seems like a good enough stop-gap.